database/mortgage_rate.py

```diff
@@ -0,0 +1,37 @@
+# range_300_619 = {
+#     "credit_score_floor": "620",
+#     "credit_score_cieling": "639",
+#     "rate": 
+# "This one is special": "And needs something informational"
```

Comment:
Are we putting in a condition for users with credit scores below 620? Do we assign the max mortgage rate? I would be wary about deterring a user with a "low" credit score